pulp_rpm/app/access_policy.py

```diff
@@ -0,0 +1,50 @@
+from logging import getLogger
+
+from pulpcore.plugin.models import RepositoryVersion
+from pulpcore.plugin.viewsets import NamedModelViewSet
+from pulpcore.plugin.access_policy import AccessPolicyFromDB
+
+from pulp_rpm.app.models.repository import RpmRepository
+
+_logger = getLogger(__name__)
+
+
+class CopyAccessPolicy(AccessPolicyFromDB):
```

Comment:
We must not use custom access policies anymore. This contradicts that we can configure them. Instead write global access conditions:
https://github.com/pulp/pulpcore/blob/main/docs/plugins/plugin-writer/concepts/rbac/access_policy.rst#custom-permission-checks
example: https://github.com/pulp/pulp_container/commit/481e8bdd363b5d4dfc617db359caebd5a2bbd8ea

Reply:
@mdellweg you for got to mention that if using global access conditions then the pulpcore requirement needs to be bumped to 3.19
@pavelpicka FYI